pkg/serializer/split/split.go

```diff
@@ -163,3 +169,14 @@ func marshal(m marshaler.Marshaler, mType MarshalType) ([]byte, error) {
 		return m.MarshalJSON()
 	}
 }
+
+func handleDroppedPayloads(droppedPayloads ...[]marshaler.Marshaler) {
```

Comment:
No need for `droppedPayloads ...[]marshaler.Marshaler` anymore ;)

Reply:
Oh yes, I wonder why go doesn't give an error in the case of unused function arguments like unused variable

Reply:
My guess would be to allow satisfying interfaces without using the arguments imposed by the interface in the function

Reply:
Oh yes, probably :)